integration-test/src/test/java/org/ehcache/integration/ThreadPoolsTest.java

```diff
@@ -35,13 +37,13 @@ public void testNoDefaultThreadPoolFails() throws Exception {
     PooledExecutionServiceConfiguration executionServiceConfiguration = new PooledExecutionServiceConfiguration();
     executionServiceConfiguration.addPool("foo", 2, 4);
     CacheManager cacheManager = CacheManagerBuilder.newCacheManagerBuilder()
-        .using(executionServiceConfiguration)
```

Comment:
Try to avoid disturbing existing indentation unless it's very obviously wrong.

Reply:
applied Java indentation to file through inteliJ.
May need different code style used by project